src/Congo.RazorPages/Pages/Products.cshtml.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Congo.RazorPages.Models;
+using Congo.RazorPages.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Congo.RazorPages.Pages
+{
+    public class ProductsModel : PageModel
+    {
+        public static List<Product> SampleData;
+        
+        public void OnGet([FromServices] IProductsService productsService)
```

Comment:
We'd likely want to reuse this service down the line, when we push a button or something. So it would make more sense to make a field for this and set it in the constructor.